modules/input/react/README.md

```diff
@@ -0,0 +1,119 @@
+# Canvas Kit Input
+
+Canvas-styled React components.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
```

Comment:
`yarn add @workday/canvas-kit-react-input`

Reply:
`yarn add @workday/canvas-kit-react-input` is added on line 14